src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,98 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+import { normalizeMovieData } from '../../helpers';
+
+interface Props {
+  setMovies: React.Dispatch<React.SetStateAction<Movie[]>>;
+  movies: Movie[];
+}
+
+export const FindMovie: React.FC<Props> = ({
+  setMovies,
+  movies,
+}) => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [movieData, setMovieData] = useState<MovieData | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+
+  const handleChangeSearchInput = (
```

Comment:
usually we wrap callbacks into the `useCallback` hooks. This way you have it in your code now - we re-create this callback on each render, even if we don't have any reason for that